catalog/core/catalog-core-solr/pom.xml

```diff
@@ -128,7 +123,7 @@
                             <include>ddf.platform.solr:solr-xpath</include>
                             <include>org.apache.lucene:lucene-core</include>
                             <include>ddf.catalog.core:catalog-core-api-impl</include>
-                            <include>com.spatial4j:spatial4j</include>
+                            <include>org.locationtech.spatial4j:spatial4j</include>
```

Comment:
Do you need to add `org.locationtech.spatial4j` as a dependency? not sure where maven shade is getting it's version and such without the explicit declaration.